File: app/src/main/java/com/tstudioz/fax/fme/database/KolegijTjedan.java
package com.tstudioz.fax.fme.database;

import io.realm.RealmList;
import io.realm.RealmObject;

/**
 * Created by amarthus on 09-May-17.
 */

public class KolegijTjedan extends RealmObject{


    public String opis;
    public String tjedan;
    public RealmList<Materijali> materijali;



    public void setMaterijali(RealmList<Materijali> materijali) {
        this.materijali = materijali;
    }

    public void setOpis(String opis) {
        this.opis = opis;
    }

    public void setTjedan(String tjedan) {
        this.tjedan = tjedan;
    }


    public RealmList<Materijali> getMaterijali() {
        return materijali;
    }

    public String getOpis() {
        return opis;
    }

    public String getTjedan() {
        return tjedan;
    }

}

File: app/src/main/java/com/tstudioz/fax/fme/database/KolegijTjedan.kt
package com.tstudioz.fax.fme.database

import io.realm.kotlin.types.RealmList
import io.realm.kotlin.types.RealmObject

open class KolegijTjedan : RealmObject {

    var index = 0
    var opis: String? = null
    var tjedan: String? = null
    var materijali: RealmList<Materijal>? = null

}
